gulp/server.js

```diff
@@ -31,12 +31,13 @@ function browserSyncInit(baseDir, browser) {
      */
     server.middleware = [
         proxyMiddleware('/rest', {
-            target: 'http://localhost:9090/',
+            target: 'https://localhost:9090/',
```

Comment:
This file should only have the `https` settings if the locally running `service` and `saml-sp` are running on https. I'm not sure if that's the way it's set up. If it is, then this is fine; if not I think this file should revert